aws/modules/database/parameter_group.tf

```diff
@@ -0,0 +1,10 @@
+resource "aws_db_parameter_group" "db" {
+  name   = "${var.id}"
+  family = "postgres9.5"
+
+  parameter {
+    name = "log_hostname"
+    value = "0"
```

Comment:
Possibly. It's represented in the AWS console as `0` and `1` but I'll see if deals with other values.